.github/workflows/cron.yml

```diff
@@ -0,0 +1,27 @@
+name: "Hourly Build & Deploy to Netlify"
```

Comment:
What the difference between this and `.github/workflows/main.yml` (which also has a `cron: '0 * * * *'` line)?

Reply:
oh sorry, i was suppose to remove cron from `main.yml`. i'll fix it.

Reply:
I mean what's the difference? They look very similar. Do we need both?

Reply:
in cron.yml I also remove `github-token` to not add comments every hour. it makes listening to notifications on this github repo almost useless. this is the main reason for the split.